### PR TITLE
Magento_Checkout: avoid using deprecated escape* methods from Abstrac…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests EE, Static Tests, Unit Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests EE

File: app/code/Magento/Checkout/view/frontend/templates/button.phtml [hunk 1]
```diff
@@ -4,14 +4,19 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Onepage\Success */ ?>
+<?php
+/**
+ * @var $block \Magento\Checkout\Block\Onepage\Success
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/button.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Onepage\Success $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/form.phtml [hunk 2]
```diff
@@ -6,15 +6,18 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/**  @var $block \Magento\Checkout\Block\Cart\Grid */
+/**
+ * @var $block \Magento\Checkout\Block\Cart\Grid
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/form.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Grid $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/configure/updatecart.phtml [hunk 3]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Catalog\Block\Product\View */
+/**
+ * @var $block \Magento\Catalog\Block\Product\View
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/item/configure/updatecart.phtml]:
```suggestion
 * @var \Magento\Catalog\Block\Product\View $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml [hunk 4]
```diff
@@ -7,7 +7,10 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate
 // phpcs:disable Magento2.Files.LineLength.MaxExceeded
 
-/** @var $block \Magento\Checkout\Block\Cart\Item\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Cart\Item\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Item\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/price/sidebar.phtml [hunk 5]
```diff
@@ -6,13 +6,16 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/item/price/sidebar.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/renderer/actions/edit.phtml [hunk 6]
```diff
@@ -4,12 +4,15 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Cart\Item\Renderer\Actions\Edit */
+/**
+ * @var $block \Magento\Checkout\Block\Cart\Item\Renderer\Actions\Edit
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/item/renderer/actions/edit.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Item\Renderer\Actions\Edit $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/renderer/actions/remove.phtml [hunk 7]
```diff
@@ -4,13 +4,16 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Cart\Item\Renderer\Actions\Remove */
+/**
+ * @var $block \Magento\Checkout\Block\Cart\Item\Renderer\Actions\Remove
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/item/renderer/actions/remove.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Item\Renderer\Actions\Remove $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/minicart.phtml [hunk 8]
```diff
@@ -4,14 +4,17 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Cart\Sidebar */
+/**
+ * @var $block \Magento\Checkout\Block\Cart\Sidebar
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/minicart.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Sidebar $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/noItems.phtml [hunk 9]
```diff
@@ -4,15 +4,18 @@
  * See COPYING.txt for license details.
  */
 
-/**  @var $block \Magento\Checkout\Block\Cart */
+/**
+ * @var $block \Magento\Checkout\Block\Cart
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/noItems.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml [hunk 10]
```diff
@@ -5,7 +5,12 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Cart\Shipping */ ?>
+<?php
+/**
+ * @var $block \Magento\Checkout\Block\Cart\Shipping
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Shipping $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/item/price/row.phtml [hunk 11]
```diff
@@ -6,13 +6,16 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/item/price/row.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/item/price/unit.phtml [hunk 12]
```diff
@@ -6,13 +6,16 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/item/price/unit.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage.phtml [hunk 13]
```diff
@@ -4,15 +4,18 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Onepage */
+/**
+ * @var $block \Magento\Checkout\Block\Onepage
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Onepage $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/failure.phtml [hunk 14]
```diff
@@ -4,16 +4,19 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Onepage\Failure */
+/**
+ * @var $block \Magento\Checkout\Block\Onepage\Failure
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/failure.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Onepage\Failure $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/link.phtml [hunk 15]
```diff
@@ -4,21 +4,24 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Onepage\Link */
+/**
+ * @var $block \Magento\Checkout\Block\Onepage\Link
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/link.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Onepage\Link $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/review/item.phtml [hunk 16]
```diff
@@ -6,26 +6,29 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block Magento\Checkout\Block\Cart\Item\Renderer */
+/**
+ * @var $block Magento\Checkout\Block\Cart\Item\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/review/item.phtml]:
```suggestion
 * @var Magento\Checkout\Block\Cart\Item\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/row_excl_tax.phtml [hunk 17]
```diff
@@ -6,12 +6,15 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/row_excl_tax.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/row_incl_tax.phtml [hunk 18]
```diff
@@ -6,13 +6,16 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/row_incl_tax.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/unit_excl_tax.phtml [hunk 19]
```diff
@@ -6,12 +6,15 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/unit_excl_tax.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/unit_incl_tax.phtml [hunk 20]
```diff
@@ -6,13 +6,16 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
 
-/** @var $block \Magento\Checkout\Block\Item\Price\Renderer */
+/**
+ * @var $block \Magento\Checkout\Block\Item\Price\Renderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage/review/item/price/unit_incl_tax.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Item\Price\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/registration.phtml [hunk 21]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Registration */
+/**
+ * @var $block \Magento\Checkout\Block\Registration
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/registration.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Registration $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml [hunk 22]
```diff
@@ -5,7 +5,12 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Shipping\Price */ ?>
+<?php
+/**
+ * @var $block \Magento\Checkout\Block\Shipping\Price
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Shipping\Price $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/success.phtml [hunk 23]
```diff
@@ -5,22 +5,27 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Onepage\Success */ ?>
+<?php
+/**
+ * @var $block \Magento\Checkout\Block\Onepage\Success
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/success.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Onepage\Success $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Checkout/view/frontend/templates/total/default.phtml [hunk 24]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Total\DefaultTotal */
+/**
+ * @var $block \Magento\Checkout\Block\Total\DefaultTotal
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/total/default.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Total\DefaultTotal $block
 * @var \Magento\Framework\Escaper $escaper
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Checkout/view/frontend/templates/button.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/button.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/form.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/form.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/configure/updatecart.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/item/configure/updatecart.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/renderer/actions/edit.phtml (hunk 6 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/item/renderer/actions/edit.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/noItems.phtml (hunk 9 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/noItems.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/minicart.phtml [hunk 25]
```diff
@@ -4,14 +4,17 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Cart\Sidebar */
+/**
+ * @var \Magento\Checkout\Block\Cart\Sidebar $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/minicart.phtml]:
```suggestion
/**
 * @var \Magento\Checkout\Block\Cart\Sidebar $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Checkout/view/frontend/templates/button.phtml [hunk 26]
```diff
@@ -4,14 +4,19 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Onepage\Success */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/button.phtml]: Please remove not needed close and open PHP tag

File: app/code/Magento/Checkout/view/frontend/templates/button.phtml [hunk 27]
```diff
@@ -4,14 +4,19 @@
...
+/**
+ * @var \Magento\Checkout\Block\Onepage\Success $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
+?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/button.phtml:12]: Seems like this empty line isn't needed here
```suggestion
```

File: app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml [hunk 28]
```diff
@@ -5,7 +5,12 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Cart\Shipping */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml]: please remove not needed close and open PHP tags here

File: app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml [hunk 29]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var \Magento\Checkout\Block\Cart\Shipping $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
+?>
 <?php /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */ ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml]:
```suggestion
/**
 * @var \Magento\Checkout\Block\Cart\Shipping $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
?>
```

File: app/code/Magento/Checkout/view/frontend/templates/onepage.phtml [hunk 30]
```diff
@@ -4,15 +4,18 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Onepage */
+/**
+ * @var \Magento\Checkout\Block\Onepage $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/onepage.phtml]:
```suggestion
/**
 * @var \Magento\Checkout\Block\Onepage $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml [hunk 31]
```diff
@@ -5,7 +5,12 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Shipping\Price */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml]: Please remove here not needed closing and opening PHP tags.

File: app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml [hunk 32]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var \Magento\Checkout\Block\Shipping\Price $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
+?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml:12]: Please remove here not needed empty line
```suggestion
```

File: app/code/Magento/Checkout/view/frontend/templates/success.phtml [hunk 33]
```diff
@@ -5,22 +5,27 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Onepage\Success */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/success.phtml]: Please remove here not needed closing and opening PHP tags.

File: app/code/Magento/Checkout/view/frontend/templates/total/default.phtml [hunk 34]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Total\DefaultTotal */
+/**
+ * @var \Magento\Checkout\Block\Total\DefaultTotal $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
 ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Checkout/view/frontend/templates/total/default.phtml]: Please remove here not needed closing and opening PHP tags and combine PHP doc blocks to single block.

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Checkout/view/frontend/templates/cart/minicart.phtml (hunk 25 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/minicart.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/button.phtml (hunk 26 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/button.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/button.phtml (hunk 27 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/button.phtml:12]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml (hunk 28 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml (hunk 29 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/cart/shipping.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/onepage.phtml (hunk 30 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/onepage.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml (hunk 31 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml (hunk 32 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml:12]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/success.phtml (hunk 33 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/success.phtml]: fixed

File: app/code/Magento/Checkout/view/frontend/templates/total/default.phtml (hunk 34 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Checkout/view/frontend/templates/total/default.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Unit Tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Pull request closed without merging**